### PR TITLE
Add popup hover for seeing countries in template

### PR DESCRIPTION
**Pull request by kumar-ish**
Looks like this:

image

A little ugly on mobile and not-great for templates with a ton of member countries, will fix later.

### CODE REVIEW

**Comment by kumar-ish** [comment]
lol it's not terrible and it works, if you have any suggestions I can change it accordingly @MaxwellBo

**Comment by MaxwellBo** [comment]
Did you see my review comments?

**Comment by kumar-ish** [comment]
Hmm, I can't see anything about it. It was a draft before anyways, so not completely sure where they are?

**Comment by MaxwellBo** [comment]
@kumar-ish sorry about that, it's been pending this entire time (18 days)

File: src/components/Onboard.tsx [hunk 1]
```diff
@@ -116,6 +116,31 @@ export default class Onboard extends React.Component<Props, State> {
...
+      if (props.template) {
+        countryRows = TEMPLATE_TO_MEMBERS[props.template]
+          .map(member => (<Table.Row>
+                            <Table.Cell>{member.name}</Table.Cell>
+                          </Table.Row>)
+          );
+      }
+
+      return (props.template ?
+              (<Table basic='very' padded={false} celled={false}>
```
Comment (MaxwellBo) [inline src/components/Onboard.tsx:134]: What does this look like as a `<ul>` rather than a table?

**Comment by MaxwellBo** [comment]
withflags2
@kumar-ish made some changes, let me know what you think

**Pull request merged**